Jenkinsfile

```diff
@@ -13,7 +13,7 @@ pipeline {
             steps {
                 sh '''
                 rm -rf test-reports/TEST-*.xml
-                jbsub -wait -out ccc_log.txt -queue x86_1h -mem 40g -cores "4+1" ./run_all_unit_tests.sh 11.6 /dccstor/fuse_med_ml/cicd/envs/
+                jbsub -wait -out ccc_log.txt -queue x86_1h -mem 40g -cores "10+1" ./run_all_unit_tests.sh 11.6 /dccstor/fuse_med_ml/cicd/envs/
```

Comment:
We though maybe to add also:
-require v100

Reply:
Correct.
I accidentally added `-require *a100*` to ISIC's PR, so I'm updating both now.